doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib

```diff
@@ -59,6 +113,11 @@ NGSI version 2 uses camel case (camelCase) syntax for naming properties and rela
 by the API. When  referring to URIs, as part of HATEOAS patterns, and to mark them appropriately,
 the suffix `_url` is added.
 
+## Reference Implementations
+
+* NGISv2 servers
```

Comment:
NGSIv2 Context Brokers

Reply:
Fixed in 0d64f7138b3462067571346ea6d71352131bfef0